app/javascript/components/gtl/DataTable.jsx

```diff
@@ -12,6 +12,14 @@ const getNodeIconType = (row, columnKey) =>
 
 const isFilteredBy = (settings, column) => settings.sort_col === column.col_idx;
 
+const arrayRemove = (array) => {
+  console.log("unSelectAll arrayRemove dataTable")
+  for (var i=0; i<array.length; i++) {
```

Comment:
try to use array for each (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) for cleaner code.